pyopensprinkler/__init__.py

```diff
@@ -340,6 +340,15 @@ def run_once_program(self, station_times):
         (_, content) = self.request("/cr", None, f"t={t}")
         return content["result"]
 
+    def set_password(self, password):
```

Comment:
I thought i could set the initial password using this but couldn't so this is not needed but left it in anyway